_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -93463,7 +93461,6 @@
 "dBS" = (
 /obj/structure/table/wood/poker,
 /obj/item/reagent_containers/food/drinks/shaker,
-/obj/item/reagent_containers/glass/beaker/cryoxadone,
```

Comment:
there's usually at least one spare (as in, not being used in a cryo pod) cryo beaker out in the open in medbay near shiftstart, and I don't see people grab that most rounds; why does this maint one have to be removed?

Reply:
doesn't make sense for one to be in bar

Reply:
you can make certain drinks and synthmeat out of it, no?